src/lib/trips.js

```diff
@@ -91,16 +91,24 @@ const getFeatureMode = feature => {
   return manualMode || sensedMode
 }
 
+const getFeatureModes = feature => {
+  return feature.features.map(feature => getFeatureMode(feature))
+}
+
+const tripsSortedByDistance = (a, b) => {
+  return get(a, 'features[0].properties.distance') >
+    get(b, 'features[0].properties.distance')
+    ? -1
+    : 1
```

Comment:
personally I would have extract a function:

`const getDistance = x => get(x, 'features[0].properties.distance')`

and then 
```
const tripsSortedByDistance = (a, b) => getDistance(a) < getDistance(b)
```

let's test the method and check i'm not wrong 🙏  (personally I would have exported tripsSortedByDistance for test purpose)

Reply:
on peut plus simplement utiliser sortBy de lodash aussi, c'est plus simple à comprendre

Reply:
> personally I would have extract a function

👍 

> personally I would have exported tripsSortedByDistance for test purpose

Elle est couverte par le test de `getModesSortedByDistance`